test/CadlRanchProjects/azure/core/lro/standard/src/Generated/StandardClient.cs

```diff
@@ -437,5 +438,11 @@ internal static RequestContext FromCancellationToken(CancellationToken cancellat
         private static ResponseClassifier ResponseClassifier200201 => _responseClassifier200201 ??= new StatusCodeClassifier(stackalloc ushort[] { 200, 201 });
         private static ResponseClassifier _responseClassifier202;
         private static ResponseClassifier ResponseClassifier202 => _responseClassifier202 ??= new StatusCodeClassifier(stackalloc ushort[] { 202 });
+
+        private ExportedUser FetchExportedUserFromResourceOperationStatusUserExportedUserError(Response response)
```

Comment:
Conversion method is named after pattern `Fetch{ReturnType}From{RawResponseType`. For this test case, the name is pretty long, because `RawResponseType` is a typespec core template.